app/assets/stylesheets/juntos_bootstrap/components/_flash_messages.scss

```diff
@@ -0,0 +1,48 @@
+.flash {
+  p {
+    margin-left: 10px;
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+  }
+}
+
+.icon-close {
+  -o-transition: all 500ms ease;
+  -webkit-transition: all 500ms ease;
+  display: block;
+  float: right;
+  margin-left: 3px;
+  opacity: 0.5;
+  padding: 0px;
+  transition: all 500ms ease;
+}
+.icon-close:hover {
+  opacity: 1;
+}
+
+
+@media (max-width: 991px) {
```

Comment:
Prefer mobile-first using `min-width` when possible, using `min-width` is way more maintainable than using `max-width`.